OrPinhas_and_AvivShabtay_Project/OrPinhas_and_AvivShabtay/OrPinhas_and_AvivShabtay/Person.h

```diff
@@ -12,15 +12,15 @@ class Person
 public:
 	Person(int age, char* strName)
 	{
-		this-> age = age
-		name = new char[strlen(strName) + 1];
+		this->age = age
+			name = new char[strlen(strName) + 1];
 		strcpy(name, strName);
 	};
 
 	Person(Person& person)
 	{
 		age = person.age
-		name = new char[strlen(person.name) + 1];
+			name = new char[strlen(person.name) + 1];
```

Comment:
Remove the blank spaces before the name variable.
Add this before class member in order to distinguish between parameter and the class member himself.